### PR TITLE
Casper Livenet updates to match 1.5.1

### PR DESCRIPTION
**Pull request by zie1ony**
In addition I have changed the `constructor` arg to `odra_cfg_constructor` to match other constructor args.

### CODE REVIEW

File: odra-casper/codegen/src/call_method.rs [hunk 1]
```diff
@@ -81,7 +81,7 @@ impl ToTokens for CallMethod {
 
                 #entry_points
 
-                #[allow(dead_code)]
+                #[allow(unused_variables)]
                 let contract_package_hash = odra::casper::utils::install_contract(entry_points, schemas);
```
Comment (kubaplas) [inline odra-casper/codegen/src/call_method.rs:85]: let _contract_package_hash?

File: odra-casper/codegen/src/lib.rs [hunk 2]
```diff
@@ -203,7 +203,7 @@ mod tests {
                         odra::casper::casper_types::EntryPointAccess::Public,
                         odra::casper::casper_types::EntryPointType::Contract,
                     ));
-                    #[allow(dead_code)]
+                    #[allow(unused_variables)]
                     let contract_package_hash = odra::casper::utils::install_contract(entry_points, schemas);
```
Comment (kubaplas) [inline odra-casper/codegen/src/lib.rs:207]: let _contract_package_hash =

Comment (kubaplas) [reply odra-casper/codegen/src/lib.rs:207]: Or just call it without assignement

**Review by kubaplas — APPROVED** [review]

File: odra-casper/codegen/src/call_method.rs (hunk 1 above)
Comment (zie1ony) [reply odra-casper/codegen/src/call_method.rs:85]: There is optional code that could be generated after line 85, that is using `contract_package_hash` variable.

**Pull request merged**